### PR TITLE
[pipelines] 1.8.0, added support for resources in init containers

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by eldada — APPROVED** [review]

File: stable/pipelines/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: pipelines
 home: https://jfrog.com/pipelines/
-version: 1.4.9
+version: 1.4.10
```
Comment (rimusz) [inline stable/pipelines/Chart.yaml:4]: Please change:
version: 1.4.10 -> 1.5.0
appVersion: 1.7.2 -> 1.8.0
so PR #1165 can be closed

**Review by rimusz — CHANGES_REQUESTED** [review]

File: stable/pipelines/Chart.yaml (hunk 1 above)
Comment (chukka) [reply stable/pipelines/Chart.yaml:4]: will do :)

Comment (chukka) [reply stable/pipelines/Chart.yaml]: Done

**Review by rimusz — APPROVED** [review]
LGTM

**Review by jatindogra — APPROVED** [review]

**Review by danielezer — APPROVED** [review]

**Pull request merged**